src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,47 @@
+import "./Footer.css"
+function Footer(){
+    return<>
+    <footer className="footer-containet">
```

Comment:
Typo
Doesn't affect the code as the CSS class is given the same name but it must be footer-container*